tools/clang/lib/SPIRV/EmitSPIRVAction.cpp

```diff
@@ -65,20 +31,61 @@ class SPIRVEmitter : public ASTConsumer {
       : theCompilerInstance(ci), outStream(*ci.getOutStream()),
         diags(ci.getDiagnostics()), theContext(), theBuilder(&theContext) {}
 
+  /// \brief Wrapper method to create an error message and report it in the
+  /// given diagnostics engine.
```

Comment:
There is no diagnostic engine given in the parameter list. Maybe something like "... in the diagnostic engine associated with this consumer"?